.ci.yaml

```diff
@@ -3206,6 +3206,30 @@ targets:
         ["devicelab", "android", "mac"]
       task_name: microbenchmarks
 
+  - name: Mac_android run_debug_test_android
```

Comment:
do we need the trailing `android` in the name since it's already part of the beginning?

Reply:
The first one is the platform and the second is the test name but yeah this seems a bit redundant. In terms of display on the dashboard, I think we only show the icon for the target platform (in this an android), so it should be clear enough even without android in the tooltip, I'd think.

Reply:
We didn't receive any push backs from the EngProd folks, so I'll move forward with this. This can easily be updated later if needed! :)